src/myq-lamp.ts

```diff
@@ -141,8 +137,6 @@ export class myQLamp extends myQAccessory {
 
     // Execute the command.
     void this.lampCommand(value);
-
-    return true;
```

Comment:
`setLampState()` only returns `true`, so removing this return (and adjusting code paths accordingly) addresses #1107.

Alternatively, one could adjust line 57's `onSet()` call to swallow the return value.